src/schema/v2/artwork/index.ts

```diff
@@ -471,12 +471,33 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
       isHangable: {
         type: GraphQLBoolean,
         resolve: (artwork) => {
-          const is3D =
-            _.includes(artwork.category, "sculpture") ||
+          const is3DCategory =
             _.includes(artwork.category, "installation") ||
-            _.includes(artwork.category, "design")
+            _.includes(artwork.category, "design") ||
+            _.includes(artwork.category, "performance") ||
+            _.includes(artwork.category, "sound") ||
+            _.includes(artwork.category, "fashion") ||
+            _.includes(artwork.category, "architecture") ||
+            _.includes(artwork.category, "books_and_portfolios") ||
+            _.includes(artwork.category, "sound") ||
+            _.includes(artwork.category, "jewelry") ||
+            _.includes(artwork.category, "other")
 
-          return !is3D && isTwoDimensional(artwork) && !isTooBig(artwork)
+          const is2DCategory =
+            _.includes(artwork.category, "print") ||
+            _.includes(
+              artwork.category,
+              "drawing_collage_other_work_on_paper"
+            ) ||
+            _.includes(artwork.category, "painting") ||
+            _.includes(artwork.category, "photography") ||
+            _.includes(artwork.category, "posters") ||
+            _.includes(artwork.category, "work_on_paper")
```

Comment:
There's a duplicate `sound`, but also maybe this can be less repetitive? Something like:
```
const categories3D = ["installation", "design", "performance", "sound", "fashion", "architecture", "books_and_portfolios", "jewelry", "other"]
const categories 2D = ["print", "drawing_collage_other_work_on_paper", "painting", "photography", "posters", "work_on_paper"]

const areIncluded = (category) => _.includes(artwork.category, category) 

const is3DCategory = categories3D.some(areIncluded)
const is2DCategory = categories2D.some(areIncluded)
```
I'm sure there are a million ways of simplifying this, but definitely non-blocking and up to you.

Reply:
Totally forgot about this PR, my bad! Making changes now